### PR TITLE
Updated config.json as per online editor configuration guidelines

### PR DESCRIPTION
**Pull request by hsadia538**
Fixes: #23 by updating the `indent_size`
@lpil @ErikSchierboom

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
   "version": 3,
   "online_editor": {
     "indent_style": "space",
-    "indent_size": 2,
+    "indent_size": 4,
```
Comment (lpil) [inline config.json:16]: This should be 2!

https://github.com/gleam-lang/stdlib/blob/b5d19abb14c577d86fa694e6db1548ceabb3ad43/src/gleam/result.gleam#L18

Comment (hsadia538) [reply config.json]: @lpil

**Comment by ErikSchierboom** [comment]
There are no changes in this PR AFAICT.

**Comment by lpil** [comment]
Looks like we're all good then!

**Comment by hsadia538** [comment]
@ErikSchierboom  closing this pr, please mark the v3 tracker

**Pull request closed without merging**